### PR TITLE
Finish org move

### PR DESCRIPTION
**Pull request by Changaco**
Completes #547.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,9 +22,7 @@
               , 'Topic :: Internet :: WWW/HTTP :: WSGI :: Application'
                ]
 
-setup( author = 'Gratipay, LLC'
-     , author_email = 'support@gratipay.com'
```
Comment (whit537) [inline setup.py]: Are you sure we can get away with no author?

> Either the author or the maintainer must be identified.

https://docs.python.org/2/distutils/setupscript.html#additional-meta-data

File: README.rst [hunk 2]
```diff
@@ -14,13 +14,13 @@ And we test against 32- and 64-bit Python 2.7 on Windows: |appveyor|
...
 .. _PyPI: https://pypi.python.org/pypi/aspen
-.. |travis| image:: https://img.shields.io/travis/gratipay/aspen.py/master.svg
-   :target: https://travis-ci.org/gratipay/aspen.py
+.. |travis| image:: https://img.shields.io/travis/AspenWeb/aspen.py/master.svg
+   :target: https://travis-ci.org/AspenWeb/aspen.py
    :alt: Linux build status
-.. |appveyor| image:: https://img.shields.io/appveyor/ci/gratipay/aspen-py/master.svg
-   :target: https://ci.appveyor.com/project/gratipay/aspen-py
+.. |appveyor| image:: https://img.shields.io/appveyor/ci/Changaco/aspen-py/master.svg
+   :target: https://ci.appveyor.com/project/Changaco/aspen-py
```
Comment (whit537) [inline README.rst]: I suppose this works, but why not AspenWeb here?

File: setup.py (hunk 1 above)
Comment (Changaco) [inline setup.py]: `python2 setup.py sdist bdist_wheel` works fine, the metadata files just have `UNKNOWN` for the author field.

File: README.rst (hunk 2 above)
Comment (Changaco) [inline README.rst]: Aspen would need its own AppVeyor account for that, AFAICT. Do we have an email address we can use to create an account for Aspen?

File: setup.py (hunk 1 above)
Comment (whit537) [inline setup.py]: That seems suboptimal, because Aspen has authors.

File: README.rst (hunk 2 above)
Comment (whit537) [inline README.rst]: Maybe we want mail `@aspen.io` after all.

File: setup.py (hunk 1 above)
Comment (Changaco) [inline setup.py]: Replaced with `'Chad Whitacre et al.'`.

File: README.rst (hunk 2 above)
Comment (whit537) [inline README.rst]: Email reticketed as https://github.com/AspenWeb/salon/issues/12.

File: setup.py (hunk 1 above)
Comment (whit537) [inline setup.py]: Okay, `author_email` reticketed as https://github.com/AspenWeb/salon/issues/12.

**Comment by Changaco** [comment]
Email address is up, so I guess we can move forward with this now by using it in `setup.py` and to create an AppVeyor account.

**Comment by Changaco** [comment]
Ready for review.

**Comment by whit537** [comment]
!m @Changaco

**Pull request merged**